### ISSUE TITLE
How do I use it `tailwind-merge`?

### ISSUE THREAD

**Issue by DonHuskini** [fgnass/classname-variants#8]
Hi, awesome library. Thanks! Just would like to know if it's possible to use it together with `tailwind-merge` so I can override classes if necessary.

**Comment by DonHuskini** [comment]
Found a solution. Don't know if it's the best but looks like it's working.
```tsx
export const Badge = ({ children, ...props }: BadgeProps) => {
  return (
    <div {...badgeProps(props)} className={cnMerge(badgeProps(props).className, props.className)}>
      {children}
    </div>
  )
}
```

**Comment by fgnass** [comment]
Support for using custom merge strategies landed in v1.5.0.

See https://github.com/fgnass/classname-variants?tab=readme-ov-file#using-a-custom-strategy-to-combine-class-names
